torch/_export/non_strict_utils.py

```diff
@@ -446,6 +447,54 @@ def inner(m: torch.nn.Module, prefix_atoms: List[str], constants):
     return constants
 
 
+@contextlib.contextmanager
+def _enable_module_inputs(
```

Comment:
maybe a dumb nit: I parsed this name as "inputs to the exported program module", which is wrong, since it's really "inputs to the exported program that are themselves nn.modules"

name bikeshedding.... `_enable_graph_inputs_of_type_nn_module`?